source/gameengine/GameLogic/SCA_PropertySensor.cpp

```diff
@@ -113,6 +113,9 @@ SCA_PropertySensor::~SCA_PropertySensor()
 
 bool SCA_PropertySensor::Evaluate()
 {
+	if (!(strlen(m_checkpropval.c_str()))) {
```

Comment:
evaluate is the function called each frame to see if the sensor send true pulse or not. I think it is better to stop at the beginning of evaluate